packages/workbox-webpack-plugin/src/inject-manifest.js

```diff
@@ -140,6 +149,13 @@ ${originalSWString}
 `;
 
     const relSwDest = relativeToOutputPath(compilation, this.config.swDest);
+    /**
+     * Delete if this file already exist,
+     * this is the case where source is also a webpack asset.
+     */
+    if (compilation.assets[relSwDest]) {
+      delete compilation.assets[relSwDest];
```

Comment:
Line 159 assigns a value to `compilation.assets[relSwDest]`, which should overwrite anything that's already there, so I'm not clear on what this `delete` accomplishes. Can you clarify?

Reply:
I saw that simply overriding was somehow not overriding the content, so I added an additional delete statement which did the trick

Reply:
That feels kind of magic. @developit, have you ever seen anything like that?

(Also, I'm not sure if there's a reason to wrap this in an `if()`, since the `delete` will be a no-op if the property doesn't exist.)

Reply:
Weird, I've never had to do this. @prateekbh any chance it was some other issue? I can't think of why this would happen.

Reply:
i'll re-check

Reply:
well yeah it works now.. dont know what was the problem earlier..